include/exiv2/jpgimage.hpp

```diff
@@ -104,12 +104,10 @@ class EXIV2API JpegBase : public Image {
 
   //! @name NOT implemented
   //@{
-  //! Default constructor.
-  JpegBase() = delete;
```

Comment:
Same comment in all the default constructors here. We should not allow the creation of a instance via the default constructor.